discovery-provider/plugins/pedalboard/.eslintrc.cjs

```diff
@@ -0,0 +1,15 @@
+/* eslint-env node */
```

Comment:
nit, i think we just use .js for all cjs things. for consistency, probs good to keep it that way.

We should also try to steal our configs from somewhere like i recently added to eth-contracts https://github.com/AudiusProject/audius-protocol/blob/main/eth-contracts/.eslintrc.js
 which includes prettier for formatting (no semis, etc.)